docs/apps/dozzle.md

```diff
@@ -22,4 +22,22 @@ sb install dozzle
 
 ### 3. Setup
 
-- [:octicons-link-16: Documentation](https://github.com/amir20/dozzle){: .header-icons }
+To view log files that are NOT written to sysout or syserr, use the following to setup a basic alpine container via compose that just tails a mounted log file (in this case, Cloudplow) which then exposes it to dozzle. Adjust as needed for your circumstances.
+
+``` yaml
```

Comment:
This could benefit from some annotations on what to edit to adapt this to one's needs.

Reply:
I'll get on it!

Reply:
Sure thats fine. Want me to remove all that?

Reply:
We can probably leave the note at the bottom about starting up a docker container though?